docker-compose.yml

```diff
@@ -366,11 +366,10 @@ services:
                 ip route add default via 40.40.40.1 dev eth0.40 table 1001
                 sleep 5 && \
                 ping -c 3 6.6.6.6 -I 40.40.40.40 && ping -c 3 40.40.40.41 && \
-                ip link add link eth1 name eth1.50 type vlan id 50 && \
-                ip link set eth1.50 up && \
-                ip addr add 50.50.50.50/24 dev eth1.50 && \
+                ip addr add 50.50.50.50/24 dev eth1 && \
                 ip rule add from 50.50.50.50 lookup 1002
-                ip route add default via 50.50.50.1 dev eth1.50 table 1002
+                ip route add default via 50.50.50.1 dev eth1 table 1002
+                arp -s 50.50.50.1 aa:bb:cc:00:00:51
```

Comment:
Why do you need a static ARP entry for the GW IP on this access interface? ARP should work on all types of bridge ports.

Reply:
For every container network, there is a bridge created in the host system with veth interfaces, looks like the first IP from the network is assigned to the bridge, in our case, 50.50.50.1 is assigned to that bridge as well as on vlan50. When ARP request is sent to leaf1 without any VLAN tag, the bridge in the host system is replying with it's own MAC instead of the actual MAC of vlan50, to overcome this behavior I added the static ARP. However, I changed to use different IP now on the vlan50 now i.e 50.50.50.5 instead of 50.50.50.1, we dont need the static ARP anymore.

Reply:
removed static ARP https://github.com/opiproject/opi-evpn-bridge/pull/156/commits/ef000a4d003a030cc33cf1cfc1ac831a44a16f0f